src/main/java/uk/gov/ons/census/action/utility/FulfilmentMapper.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.ons.census.action.utility;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import org.springframework.jdbc.core.RowMapper;
+
+public class FulfilmentMapper
```

Comment:
This class won't be needed anymore if you get JPA to do the heavy lifting for you